content-service/migrations/20161129113620-channel-ids.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var Promise;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
```

Comment:
This comment doesn't mean much to me. I dunno what `dbmigrate` is or `dependency on NODE_PATH` means :) But perhaps I'm not the intended reader.